pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/model/config/ComputeAIEmbeddingsConfig.java

```diff
@@ -20,6 +20,13 @@
 
 @Getter
 public class ComputeAIEmbeddingsConfig extends StepConfig {
+
+  public enum SupportedServices {
+    OPENAI,
+    HUGGINGFACES_REST,
+    HUGGINGFACES_DJL
```

Comment:
what about HUGGINGFACE   ? (no "API", this is probably the mode that will be used most of the times, as it will provide better performances)